guides/cheat-sheets/voice.cheatmd

```diff
@@ -0,0 +1,173 @@
+# Voice Usage
+
+This cheat sheet covers basic use of the Discord Voice API through the `Nostrum.Voice` module.
+
+## Playing Audio
+{: .col-2}
+
+### Playing immediately with a `try_play` function
+
+```elixir
+def try_play(guild_id, url, type, opts \\ []) do
+  case Nostrum.Voice.play(guild_id, url, type, opts) do
+    {:error, _msg} ->
+      # Wait for handshaking to complete
+      Process.sleep(100)
+      try_play(guild_id, url, type, opts)
+
+    _ ->
+      :ok
+  end
+end
+```
+
+```elixir
+Nostrum.Voice.join_channel(guild_id, channel_id)
+try_play(guild_id, "./song.mp3", :url)
+```
+
+### Playing immediately using `Nostrum.Voice.ready?/1`
+
+```elixir
+def play_when_ready(guild_id, url, type, opts \\ []) do
+  if Nostrum.Voice.ready?(guild_id) do
+    Nostrum.Voice.play(guild_id, url, type, opts)
+  else
+    # Wait for handshaking to complete
+    Process.sleep(25)
+    play_when_ready(guild_id, url, type, opts)
+  end
+end
+```
+
+```elixir
+Nostrum.Voice.join_channel(guild_id, channel_id)
+play_when_ready(guild_id, "https://youtu.be/b4RJ-QGOtw4", :ytdl)
+```
+
+### Playing immediately by using events
+
+```elixir
+alias Nostrum.Struct.Event.VoiceReady
+alias Nostrum.Voice
+```
+
+In your consumer module
+
+```elixir
+def handle_event({:VOICE_READY, %VoiceReady{guild_id: guild_id} = _event, _v_ws_state}) do
+  Voice.play(guild_id, "~/loud_noise.mp3", :url, volume: 10)
+end
+```
+
+Once voice connection handshaking is complete, audio will begin playing
+without the need for polling functions like `try_play/4` or `play_when_ready/4`
+as shown in the other examples
+
+```elixir
+Voice.join_channel(guild_id, channel_id)
+# Playback will start automatically when ready
+```
+
+## Audio FFmpeg options
```

Comment:
I'd remove the `alias Nostrum.Voice` from all these examples. We already say at the top of this cheat-sheet that we are using the `Nostrum.Voice` module, it makes the cheats longer than they need to be.